core/src/main/scala/cell/HandlerPool.scala

```diff
@@ -7,12 +7,13 @@ import scala.annotation.tailrec
 import scala.util.control.NonFatal
 import scala.concurrent.Await
 import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Would it be possible to use the handler pool instead? If not, then an `ExecutionContext` should be passed in to methods like `quiescentResolveCycles` using an implicit parameter, to make it more flexible.